web/src/components/core/InstallationFinished.jsx

```diff
@@ -34,15 +34,26 @@ import { useInstallerClient } from "~/context/installer";
 
 function InstallationFinished() {
   const client = useInstallerClient();
-  const onRebootAction = () => client.manager.rebootSystem();
+  const [iguana, setIguana] = useState(false);
+  const closingAction = client.manager.finishInstallation();
```

Comment:
This is throwing error for me:

```Expected `onClick` listener to be a function, instead got a value of `object` type.```

I have none expertise with react, but just by comparing what was before and we have now, should this be:

```suggestion
  const closingAction = () => client.manager.finishInstallation();
```